Rakefile

```diff
@@ -36,4 +36,4 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task :default => :test
+task default: [:test, :rubocop]
```

Comment:
I get where you're coming from, CI may fail due to rubocop...

I'd rather recommend adding a pre-push hook that runs rubocop... or documenting it

Perhaps we could add a `task ci: [:test, :rubocop]`...